src/solvers/prop/cover_goals.cpp

```diff
@@ -50,6 +50,7 @@ void cover_goalst::mark()
        prop_conv.l_get(g.condition).is_true())
     {
       g.status=goalt::statust::COVERED;
+      g.seconds=(current_time()-start_time);
```

Comment:
This is used to track the progression of coverage (i.e: at what time is each goal covered).